docs/docs/06-tools/01-cli.md

```diff
@@ -17,40 +17,46 @@ The CLI is distributed as an npm package which can be installed globally using:
 npm i -g winglang
 ```
 
-The usage is:
+Usage:
 
-```bash
-$ wing <command> <options>
+```sh
+$ wing [command] [options]
 ```
 
-## Run: `wing run` / `wing it`
+## Run: `wing run|it`
 
 You can use the `run` command (or `it`) when you want to interact with your Wing program in the
 [Wing Console](/docs/start-here/local).
 
 Usage:
 
-```
-$ wing run|it [<ENTRYPOINT.w> | <PROGRAM.wsim>]
+```sh
+$ wing run|it [entrypoint]
 ```
 
-The `run` command takes a single positional argument which can be one of:
+`[entrypoint]` is a valid entrypoint for a Wing program. An entrypoint can be either source code or compiled. If it's source code, the entrypoint refers to a file named `main.w` or ending with `.main.w`/`.test.w`, whereas if it's compiled the entrypoint refers to a directory whose name ends with `.wsim`.
 
-- `ENTRYPOINT.w` is an entrypoint for a Wing program (source code). In this case, Wing Console will
-  *watch for changes* and will automatically recompile your program when the source code change.
-  The entrypoint is optional if there's a single `.w` file in the working directory (in which case you
-  can just type `wing it` and it will run this file).
-- `PROGRAM.wsim` is the output of `wing compile -t sim`
+:::note Default Entrypoint
+
+It's possible to execute `wing run|it` without specifying any entrypoint, in which case the CLI looks for exactly one file named `main.w` or ending with `.main.w` in the current directory. If no files or more than one file are found, the CLI throws an error.
```

Comment:
This can be shortened, no need to mention the error IMO

```suggestion
By default, `wing run|it` will look for exactly one file named `main.w` or ending with `.main.w`
```